lib/client-assets.php

```diff
@@ -695,3 +704,16 @@ function gutenberg_extend_settings_custom_units( $settings ) {
 	register_block_pattern( 'core/hero-right-column', gutenberg_load_block_pattern( 'hero-right-column' ) );
 	register_block_pattern( 'core/testimonials', gutenberg_load_block_pattern( 'testimonials' ) );
 }
+
+/*
+ * Register default pattern categories if not registered in Core already.
+ */
+if ( class_exists( 'WP_Block_Pattern_Categories_Registry' ) ) {
+	register_block_pattern_category( 'text', array( 'label' => __( 'Text', 'gutenberg' ) ) );
+	register_block_pattern_category( 'hero', array( 'label' => __( 'Hero', 'gutenberg' ) ) );
```

Comment:
`_x()` would be especially useful for this case IMO. Could be very confusing for translators.